graphql/src/main/java/io/micronaut/configuration/graphql/GraphiQLController.java

```diff
@@ -15,7 +15,7 @@
  */
 package io.micronaut.configuration.graphql;
 
-import io.micronaut.configuration.graphql.apollo.ws.GraphQLApolloWsConfiguration;
+import io.micronaut.configuration.graphql.ws.apollo.GraphQLApolloWsConfiguration;
```

Comment:
is this package name necessary? did `io.micronaut.configuration.graphql.apollo.ws` package existed in MN 3?